packages/server-reason-react-ppx/cram/styles.t/input.re

```diff
@@ -1 +1,9 @@
+
 <div styles=x />;
+<div styles=?x />;
+<div className="lola" styles=x />;
+<div style={ReactDOM.Style.make(~backgroundColor="gainsboro", ())} styles=x />;
+<div className="lola" style={ReactDOM.Style.make(~backgroundColor="gainsboro", ())} styles=x />;
+<div className="lola" styles=?x />;
+<div style={ReactDOM.Style.make(~backgroundColor="gainsboro", ())} styles=?x />;
+<div className="lola" style={ReactDOM.Style.make(~backgroundColor="gainsboro", ())} styles=?x />;
```

Comment:
I belive cram tests are good enough, no need to repeat all tests as runtime (maybe keep 1 or 2) but having repetition there seems too much, thoughts?

Reply:
Yes, having 1 test like this is ok, but not all of them. That's my comment.